app/src/androidTest/java/swati4star/createpdf/ApplicationTest.java

```diff
@@ -1,8 +1,6 @@
 package swati4star.createpdf;
 
 import android.content.Context;
-import android.support.test.InstrumentationRegistry;
-import androidx.test.ext.junit.runners.AndroidJUnit4;
```

Comment:
Changes are different than what title is telling about.

Reply:
Thank you for letting me know about the problems. I'm sorry I am not a good user of GitHub :(
I closed my PRs add oped new ones. And I fixed everything you said. 
I hope you check it out soon :)
Thank you for your feedback!